src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlServerFhirModel.cs

```diff
@@ -294,18 +315,10 @@ COMMIT TRANSACTION
                         _compartmentTypeToId = compartmentTypeToId;
                     }
                 }
-
-                // Search parameter statuses are only supported in version four and higher.
-                if (_schemaInformation.Current >= 5)
-                {
-                    InitializeSearchParameterStatusInformation();
-                }
-
-                _started = true;
             }
         }
 
-        private void InitializeSearchParameterStatusInformation()
+        private void InitializeV6()
```

Comment:
>InitializeV6 [](start = 21, length = 12)

I like how descriptive function name was before renaming. It's hard mentally to remember difference between version 1 and 6 in a long run.
I would personally prefer more descriptive names to methods than `InitV1` and `InitV6` or maybe put comments on top of them which would describe difference or what they actually do.

Reply:
As mentioned above, I renamed the methods in eef506a.